test/unit/creator.js

```diff
@@ -0,0 +1,130 @@
+module.exports =  {
```

Comment:
yeah maybe, I just took what was in the tests and moved it in here :) but now we can refactor away :D